packages/components/src/DataList/components/DataListHeaderTile/SortingArrows.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./SortingArrows.css";
+import { Icon } from "../../../Icon";
+
+export function SortingArrows({ order }: { order: "asc" | "desc" }) {
+  const activeColor = "greyBlue";
```

Comment:
should be grey blue lighter. If that doesn't exist on the icon color, make it support it.

Reply:
Good catch, added it!